pkg/hosts/hosts.go

```diff
@@ -2,27 +2,49 @@ package hosts
 
 import (
 	"fmt"
+	"regexp"
 	"sort"
 	"strings"
 
 	"github.com/goodhosts/hostsfile"
 )
 
+const (
+	ClusterDomain = ".crc.testing"
```

Comment:
This can be refined later, but we may be able to hardcode the allowed `.crc.testing` names, as I'm not sure the cluster will create many of these.